logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -329,7 +329,6 @@
 
     let(:agent_args) do
       {
-        "metric.collect" => true,
```

Comment:
Why was this removed? Were metrics off by default during tests before?

Reply:
moving forward, there will not be any 'turn on/off the metrics' flag